rosbag2_performance/rosbag2_performance_benchmarking/CMakeLists.txt

```diff
@@ -36,25 +36,25 @@ if(BUILD_ROSBAG2_BENCHMARKS)
     src/result_utils.cpp
     src/results_writer.cpp)
 
-  ament_target_dependencies(writer_benchmark
-    rclcpp
-    std_msgs
-    rosbag2_compression
-    rosbag2_cpp
-    rosbag2_storage
-    yaml_cpp_vendor
+  target_link_libraries(writer_benchmark
+    rclcpp::rclcpp
+    ${std_msgs_TARGETS}
+    rosbag2_compression::rosbag2_compression
+    rosbag2_cpp::rosbag2_cpp
+    rosbag2_storage::rosbag2_storage
+    ${YAML_CPP_LIBRARIES}
   )
 
-  ament_target_dependencies(benchmark_publishers
-    rclcpp
-    rosbag2_storage
-    std_msgs
-    yaml_cpp_vendor
+  target_link_libraries(benchmark_publishers
+    rclcpp::rclcpp
+    ${std_msgs_TARGETS}
+\    rosbag2_storage::rosbag2_storage
```

Comment:
It seems typo here "`\`".

Reply:
@MichaelOrlov I fixed typo [here](https://github.com/ros2/rosbag2/pull/1202/commits/af93d5edc317a54c75824decbc2eb79396c5706c)